lib/worker-pipeline.js

```diff
@@ -69,6 +70,26 @@ class AssetComputeWorkerPipeline {
         return engine;
     }
 
+    normalizeInputOuput(input, output) {
+        // TODO: we will have special cases for beta-worker-creative 
+        // special case for sensei
+        if(output.fmt === 'machine-metadata-json') {
+            output.type = 'machine-metadata-json';
```

Comment:
In theory we could make ourselves smarter and investigate if we could use a custom mimetype ([it would contain a vnd keyword](https://en.wikipedia.org/wiki/Media_type#Vendor_tree)).